### PR TITLE
Fix constant issue and not checking the tap of brew

### PR DESCRIPTION
**Pull request by kalinstanchev**
Add check for henkrehorst php tap in brew, fix valet fix with right constant.

### CODE REVIEW

File: cli/Valet/Pecl.php [hunk 1]
```diff
@@ -313,7 +313,7 @@ function fix(){
         // Check if pear config is set correctly as per:
         // https://github.com/kabel/homebrew-core/blob/2564749d8f73e43cbb8cfc449bca4f564ac0e9e1/Formula/php%405.6.rb
         // Brew installation standard.
-        foreach (Brew::SUPPORTED_PHP_FORMULAE as $phpVersion => $brewname) {
+        foreach (PhpFpm::SUPPORTED_PHP_FORMULAE as $phpVersion => $brewname) {
             output("Checking php $phpVersion...");
 
             $pearConfigPath = "/usr/local/etc/php/$phpVersion/pear.conf";
```
Comment (Neodork) [inline cli/Valet/Pecl.php]: This one also needs to be changed to: `"/usr/local/etc/valet-php`

Comment (kalinstanchev) [reply cli/Valet/Pecl.php]: But should this be only for PHP 5.6 and few more? Or all the versions will be there?

Comment (Neodork) [reply cli/Valet/Pecl.php]: Yeah for all, we moved the config path of all valet-php versions to ensure there will be no conflict with local installs.

**Comment by Neodork** [comment]
@samgranger LGTM please review.

**Comment by kalinstanchev** [comment]
@Neodork @samgranger Could you proceed with UAT please? Thanks

**Pull request merged**